### PR TITLE
remove google_maps feature flag check

### PR DESCRIPTION
**Pull request by gfiorav**
CR @javitonino 

brings changes from 3.13.x

cc/ @javisantana @rafatower

### CODE REVIEW

**Comment by javitonino** [comment]
The feature flag is used in the views (account settings page) to show/hide the Google API keys. With the new code, this will never be shown. Maybe change the check to use  `google_maps_geocoder_enabled?`?

**Comment by gfiorav** [comment]
good point, removed them from the templates

the presenter used the geocoder check so it should be fine

**Comment by javisantana** [comment]
good catch @javitonino

**Comment by gfiorav** [comment]
Hey @javitonino thanks for looking into it with so much detail!

I have changed a few things that were wrongly displayed. Have a look and I'll merge! 

Thanks.

File: app/views/admin/client_applications/api_key.html.erb [hunk 1]
```diff
@@ -76,7 +76,7 @@
           <label class="FormAccount-label">Google Maps</label>
         </div>
         <div class="FormAccount-rowData">
-          <input type="text" value="<%= current_user.organization.present? ? current_user.organization.google_maps_key : current_user.google_maps_key %>" class="FormAccount-input FormAccount-input--long is-disabled" readonly />
+          <input type="text" value="<%= current_user.organization.present? ? current_user.organization.google_maps_private_key : current_user.google_maps_private_key %>" class="FormAccount-input FormAccount-input--long is-disabled" readonly />
```
Comment (javitonino) [inline app/views/admin/client_applications/api_key.html.erb]: Pre-existing bug?

**Comment by javitonino** [comment]
Looks good for backend. Will there be matching changes to the JS?

File: app/views/admin/client_applications/api_key.html.erb (hunk 1 above)
Comment (gfiorav) [inline app/views/admin/client_applications/api_key.html.erb]: Yes! Decided to fix it as well

**Comment by gfiorav** [comment]
Thanks for reviewing! I think the issues with Travis build is due to something else. I'll try to rebuild.

File: app/views/admin/client_applications/api_key.html.erb (hunk 1 above)
Comment (javitonino) [inline app/views/admin/client_applications/api_key.html.erb]: :palm_tree: :heart:

**Comment by gfiorav** [comment]
🍏 Here as well, merging! Thnx @javitonino!

**Pull request merged**